### ISSUE TITLE
Can't import ModSubprint.CardsToRemove from .json

### ISSUE THREAD

**Issue by d3ara1n** [stacklandsdev/mod-template#3]
```c#
// GameDataLoader.LoadModSubprint
Subprint subprint2 = new Subprint();
      subprint2.RequiredCards = ((IEnumerable<string>) subprint1.RequiredCards.Split(',', StringSplitOptions.None)).Select<string, string>((Func<string, string>) (str => str.Trim())).ToArray<string>();
      Subprint subprint3 = subprint2;
      string requiredCards = subprint1.RequiredCards;
      string[] array = requiredCards != null ? ((IEnumerable<string>) requiredCards.Split(',', StringSplitOptions.None)).Select<string, string>((Func<string, string>) (str => str.Trim())).ToArray<string>() : (string[]) null;
      subprint3.CardsToRemove = array;
      subprint2.ResultCard = subprint1.ResultCard;
      subprint2.ResultAction = subprint1.ResultAction;
      Subprint subprint4 = subprint2;
      string extraResultCards = subprint1.ExtraResultCards;
      string[] strArray = (extraResultCards != null ? ((IEnumerable<string>) extraResultCards.Split(',', StringSplitOptions.None)).Select<string, string>((Func<string, string>) (str => str.Trim())).ToArray<string>() : (string[]) null) ?? new string[0];
      subprint4.ExtraResultCards = strArray;
      subprint2.Time = subprint1.Time;
      subprint2.StatusTerm = subprint1.StatusTerm ?? "";
      subprint2.statusOverride = subprint1.StatusOverride;
      blueprint.Subprints.Add(subprint2);
```
From the above decompiled code it shows that subprint.CardsToRemove is assigned from RequiredCards, is this typo or intentional?

**Comment by Cyber28** [comment]
Yeah sorry this was a really stupid mistake on my part lol, I already fixed it and it will be in the next game update :)
Such questions would be better suited for the [Stacklands Modding Discord server](https://discord.gg/j3FjwZVyWh) or the #stacklands-modding channel in the Sokpop Discord :)

**Comment by Cyber28** [comment]
Hi! Just letting you know that this was fixed in v1.4.1.15
